src/com/cli/contact/manager/SearchPhone.java

```diff
@@ -0,0 +1,20 @@
+package com.cli.contact.manager;
+
+import com.cli.contact.model.Person;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchPhone implements Searcher {
+    @Override
+    public List<Person> meetSearch(String key, List<Person> persons) {
+        List<Person> meetPersons = new ArrayList<Person>();
+
+        for (Person person : persons) {
+            if(person.getPhoneNumber() == Integer.parseInt(key)){
+                meetPersons.add(person);
```

Comment:
@anghelitalo It is ok to search the same value in your contacts, but it will be more useful if you use the method contains, to look for matches in the string, you could also use the contains in the other searchers